domain/src/commonMain/kotlin/com/lift/bro/domain/serializers/InstantSerializer.kt

```
@@ -17,6 +17,10 @@ object InstantSerializer : KSerializer<Instant> {
     }
 
     override fun deserialize(decoder: Decoder): Instant {
-        return Instant.fromEpochMilliseconds(decoder.decodeLong())
+        try {
+            return Instant.fromEpochMilliseconds(decoder.decodeLong())
+        } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)